apps/web/pages/connections.page.tsx

```diff
@@ -136,6 +146,12 @@ function ConnectionsColumn(props: ConnectionsColumnProps) {
   const onlyIntegrationId =
     integrationsRes.data?.length === 1 ? integrationsRes.data[0]?.id : undefined
 
+  function addNewConnection() {
+    if (onlyIntegrationId) {
+      void veniceConnect.connect({id: onlyIntegrationId}, {connectWith})
```

Comment:
@tonyxiao I attempted to fix error when trying to connect to Plaid here; still see server-side error.

```
  code: 'INTERNAL_SERVER_ERROR',
  name: 'TRPCError',
  [cause]: Error: Inngest API Error: 401 Event key Not Found
```

Reply:
![CleanShot 2023-02-20 at 07 39 45@2x](https://user-images.githubusercontent.com/696842/219982456-550b0eef-8bef-4d7c-ac3f-012212947889.png)
Rob just updated the docs yesterday. You need to run `npx inngest-cli@latest dev` when developing locally.